src/components/button-icon/style.ts

```diff
@@ -0,0 +1,130 @@
+import { StyleObject } from "styletron-standard";
```

Comment:
Is there possibility to reuse Button component with overrides and not to copy all of that? Probably there are reasons for that, but imho looks weird

Reply:
I don't see reason for me to reuse Button. Because it's kinda non-trivial will be support and lead to worse maintainability of component because of some changes in button will affect this component. I want to make it simple as possible because we already took a path form base web so simplicity is the main point now for me.

Reply:
Ok, i got your point. Probably then can we create a table of common styles for buttons? Because there is a lot of common styles, and - i guess `IconButton` will always have the same kinds as `Button`. Additional styles will be written in particular components, but this we can place inside `shared` and reuse in both of them.